docker-compose.yml

```diff
@@ -38,12 +38,12 @@ services:
     volumes:
       - redis:/data
 
-  postgres:
+  db:
     env_file: docker/.env
     image: postgres:10
     restart: unless-stopped
     ports:
-      - "127.0.0.1:5432:5432"
+      - "127.0.0.1:5324:5324"
```

Comment:
Better to use `127.0.0.1:${DATABASE_PORT}`. We could do the same on redis

Reply:
It gives me the below warning when I procceded with the suggestion in 41e52ce. 
```
$ sudo docker-compose up 
WARNING: The REDIS_PORT variable is not set. Defaulting to a blank string.
WARNING: The DATABASE_PORT variable is not set. Defaulting to a blank string.
```